ieeeuoft/hackathon-template
# Added README section about styling and did minor style changes

## Overview

- Added README section on the scss files
- Rearranged parts of the scss file for further organization
- Made minor style changes to the html files (stuff I wanted to revise after using this template twice)

Minor style changes to the html files are:

1. Adding a div and class so one can change the background color of the landing, dashboard, and form pages
2. Refactored the video div on the landing to work for Youtube Videos and added classes to support responsiveness

Minor style changes in the scss files are:
1. Changed the default colors to Google's default colors, not NewHacks'
2. Imported the Roboto font in the header of `base.html` so we don't have to keep looking at ugly Times New Roman
3. Utilized the `size` function from `_variables.scss`
4. Moved the customizable classes to the top of the file so people can see it

## Steps to QA

- Compile the scss and visit each page on the event app to make sure everything looks okay
